sdk/communication/azure-communication-phonenumbers/setup.py

```diff
@@ -49,7 +49,6 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
is there a reason for this? curious

Reply:
We're dropping support for Python 3.4 and Anna told me to remove it from this setup file. Also, Python 3.5 is being dropped too in the next release, so we're keeping it for now. SMS should also do this.